### DISCUSSION TITLE
Configuration Status page for debugging configuration.

### DISCUSSION THREAD

**Discussion by jmfk** [TransformerOptimus/SuperAGI#199] [Ideas]
Configuration Status page for debugging configuration.

And potentially configure and store configurations for the system through the GUI it self removing the need for non-code users to to text setup.

**Comment by jmfk** [comment]
I guess logging of critical events to the GUI would also help debugging settings, but logging is a much bigger issue with mote use cases IMHO.

**Comment by neelayan7** [comment]
We have setup configuration(API keys and other tools related data) for Tools in the GUI.
